publications/joss/paper.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'sbpy: A Python module for small-body planetary astronomy'
+tags:
+  - python
+  - astronomy
+  - solar system
+  - planetary science
+  - asteroids
+  - comets
```

Comment:
This is probably nitpicking, but doesn't small bodies also include meteoroids, centaurs, and KBOs?  Should we tag `sbpy` to include all small body categories?